client/rpc/src/lib.rs

```diff
@@ -151,6 +151,9 @@ pub mod frontier_backend_client {
 		BE: Backend<B> + 'static,
 		BE::State: StateBackend<BlakeTwo256>,
 	{
+		let Ok(at) = client.expect_block_hash_from_id(&at) else {
+			return EthereumStorageSchema::Undefined
+		};
```

Comment:
This can be simplified by using `block_hash_from_id` in the line below since we only make use of the hash:

```rust
if let Ok(Some(hash)) = client.block_hash_from_id(&at) {
   ...
```

Reply:
Sure, opened https://github.com/paritytech/frontier/pull/997 :+1: